boot/zephyr/pm.yml

```diff
@@ -16,18 +17,20 @@ mcuboot_primary:
 mcuboot_secondary:
   share_size: [mcuboot_primary]
   placement:
-    after:
-      [mcuboot_primary]
+    align: {start: DT_FLASH_ERASE_BLOCK_SIZE}
+    after: mcuboot_primary
```

Comment:
Did syntax was changed? No square brackets required anymore?

Reply:
Correct, a single item is silently converted into a one element list now.